src/contracts/rewarder/ODefaultStakerRewards.sol

```diff
@@ -339,6 +340,21 @@ contract ODefaultStakerRewards is
         emit SetAdminFee(adminFee_);
     }
 
+    /**
+     * @dev This function should be called only once and then should be upgraded to a new implementation without this function.
```

Comment:
```suggestion
     * @dev TODO: This function should be called only once and then should be upgraded to a new implementation without this function.
```
So we remember. Should also be included in migration, together with calling initializev2 on Middleware.